web/viewer.css

```diff
@@ -1541,18 +1541,43 @@ html[dir="rtl"] .treeItemToggler::before {
   box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
 }
 
+#overlayContainer > .container > .modal {
```

Comment:
Could you please test the css settings in RTL language and see if display looks well?